assets/app/view/game/corporation.rb

```diff
@@ -52,8 +52,17 @@ def render
         children << render_abilities(abilities) if abilities&.any?
 
         if @corporation.owner
-          subchildren = (@corporation.operating_history.empty? ? [] : [render_revenue_history])
-          children << h('table.center', [h(:tr, subchildren)])
+          props = {
+            style: {
+              grid: '1fr / repeat(2, max-content)',
+              gap: '2rem',
+              justifyContent: 'center',
+            },
+          }
+
+          subchildren = (n = @game.round.entities.find_index(@corporation)) ? [render_operating_order(n)] : []
```

Comment:
%s/find_index/index/g

Reply:
it'd be cleaner if it was just 

sub_children = render_operating_order(@game.round.entities.index(@corporation))

Reply:
yep, but won’t work for one test case:

```
Assets #html 1889 renders float
     Failure/Error: context.eval(script, filename: @file)
     
     MiniRacer::RuntimeError:
       >=: undefined method `>=' for nil
```

hence: assignment ? …

Reply:
i mean obviously if you copy what i did literally, but you can write code to make it work, like using a compact or returning an empty array if nil in your function

Reply:
sure, had that, but this looked cleaner to me …
I’ll change it again.